modules/ROOT/pages/keycloak/authenticating.adoc

```diff
@@ -270,6 +270,35 @@ var result = await authService.Logout(authService.CurrentUser());
 
 NOTE: To perform *backchannel or federated logouts*, you must enable the Backchannel Logout option for the federated identity provider. More information is available in the Keycloak documentation under link:https://www.keycloak.org/docs/3.3/server_admin/topics/identity-broker/oidc.html[OICD Identity Providers, window="_blank"].
 
+== Using auth with backend
+
+To authenticate service on existing backend server users need to supply additional token:
+
+[source]
+----
+Authorization: BEARER Yourtoken
+----
+
+SDK also provides number of helpers to build authentication headers that will provide way to refresh it once token is outdated.
+
+[role="primary"]
+.Android
+****
+
+. Add the `AuthHeaderProvider` instance provided by the Aerogear SDK to the current list of providers:
++
+[source,java]
+----
+authService.getAuthHeaderProvider()
```

Comment:
Users will not use core for networking so we just showing the method to get provider. that will always get the fresh token for IOS and Android.